src/utils/command.js

```diff
@@ -264,12 +266,12 @@ class BaseCommand extends TrackedCommand {
    * @param  {string} - [tokenFromFlag] - value passed in by CLI flag
    * @return {Promise<[string, string]>} - Promise containing tokenValue & location of resolved Netlify API token
    */
-  getConfigToken(tokenFromFlag) {
+  static getConfigToken(tokenFromFlag) {
```

Comment:
Seems like we can remove `getConfigToken` in favor of `getToken` altogether.

Reply:
This changed has been done and tested.